### ISSUE TITLE
Add Linting

### ISSUE THREAD

**Issue by bhavik001** [lovasoa/whitebophir#293]
Hello!
I need to implement linting feature to this repository. Can someone assign me this issue. So, that I can start working on it.

**Comment by bhavik001** [comment]
Hello!
I was implementing the ESLint Feature and I have just try to implement with on folder that is test and I am getting the below errors. Where did `Tools` came from. There is one solution for the `Tools` I can make it as a global variable in the eslintrc.js file. Then it will be okay. The second error is `done`. This is declare but the value never used. So, the solution is to remove.

There are so many error like this. If you give me permission to edit files than I can do it.

![image](https://github.com/lovasoa/whitebophir/assets/95668553/2c1c48cc-40b1-4d18-bdee-ee42a9de2ee4)

**Comment by lovasoa** [comment]
Yes you'll need to edit the files.

Tools is defined here: https://github.com/lovasoa/whitebophir/blob/master/client-data/js/board.js#L27

**Comment by bhavik001** [comment]
Hey,
I have created a pull request please look into it and also I am creating a document which consist of the line number and error that I have disable. Also don't close this pull request after merging it.

**Comment by bhavik001** [comment]
[Remaining ESLint Error.txt](https://github.com/lovasoa/whitebophir/files/13624411/Remaining.ESLint.Error.txt)
Hey 
I have created  new pull request and the above file consist of the changes we have to do. For now I have disable ESLint for that particular line.

**Comment by lovasoa** [comment]
Hello! I don't think we can consider this ready to merge until you fix the actual eslint errors. You can't just disable lints everywhere and call it a day.

**Comment by bhavik001** [comment]
Yup, I am continue my work. Just need some help to understand some part of the code.

**Comment by lovasoa** [comment]
Sure, let me know what you need help with

**Comment by bhavik001** [comment]
Hey,
I have updated the file and also dealt with  all the errors. Please review the code and also I am working on the documentation and plugin for the contributor.

**Comment by lovasoa** [comment]
I receive the notifications from the pull request already. No need to keep posting in this issue every time you update the PR :)

**Comment by lovasoa** [comment]
The project already has a linting toolchain, but it is Biome rather than ESLint.

Current scripts in `package.json`:

```json
"lint": "biome check . --assist-enabled=false --error-on-warnings",
"format": "biome check . --write --unsafe --assist-enabled=false"
```

`npm test` also runs the Node suite, Playwright suite, and Biome lint. Introducing ESLint in parallel would add another source of formatting/lint ownership without a clear benefit, and would increase maintenance cost for contributors.

Closing this as not planned. Lint improvements should target the existing Biome setup unless there is a specific rule or capability Biome cannot cover.
